### PR TITLE
 CZECH translation update

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

**Comment by flashcode** [comment]
Merged.

File: po/cs.po [hunk 1]
```diff
@@ -10066,13 +10038,80 @@ msgid ""
...
+"Input allowed on script buffer:\n"
+"  i/r/l/L/u/h  action on script (same as keys above)\n"
+"  q            close buffer\n"
+"  $            refresh buffer\n"
+"  s:x,y        sort buffer using keys x and y (see /help script.look.sort)\n"
+"  s:           reset sort (use default sort)\n"
+"  word(s)      filter scripts: search word(s) in scripts (description, "
+"tags, ...)\n"
+"  *            remove filter\n"
+"\n"
```
Comment (flashcode) [inline po/cs.po]: I merged the pull request, but for your info, there are some lines not translated above this point.

Comment (mimi1vx) [inline po/cs.po]: yes , I known, this is why I add 'fuzzy' mark

Comment (flashcode) [inline po/cs.po]: Oh sorry, I missed the fuzzy here ;)

**Pull request closed without merging**